redfish-core/lib/ethernet.hpp

```diff
@@ -70,6 +70,7 @@ struct IPv6AddressData
     std::string address;
     std::string origin;
     uint8_t prefixLength = 0;
+    bool isActive;
```

Comment:
CI detects that this field needs to be initialized on the constructor.
```
bool isActive{};
```

Reply:
done